# -*- coding: utf-8 -*-
# Generated by Django 1.11.3 on 2017-08-16 13:51
from __future__ import unicode_literals

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('main', '0002_auto_20170811_2119'),
    ]

    operations = [
        migrations.CreateModel(
            name='WorkHistory',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('company', models.CharField(max_length=250)),
                ('at_company_from', models.DateField(null=True)),
                ('at_company_till', models.DateField(null=True)),
                ('present', models.BooleanField(default=False)),
                ('role', models.CharField(max_length=250)),
                ('description', models.TextField()),
            ],
        ),
        migrations.AlterField(
            model_name='rootuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
        migrations.AlterField(
            model_name='technologies',
            name='expertise',
            field=models.PositiveSmallIntegerField(choices=[(0, 'Proficient'), (1, 'Mediocre'), (2, 'Beginner')]),
        ),
    ]
